app/components/admin/settings/configuration_tab_component.rb

```diff
@@ -0,0 +1,39 @@
+class Admin::Settings::ConfigurationTabComponent < ApplicationComponent
+  def settings
+    %w[
+      official_level_1_name
+      official_level_2_name
+      official_level_3_name
+      official_level_4_name
+      official_level_5_name
+      max_ratio_anon_votes_on_debates
+      max_votes_for_debate_edit
+      max_votes_for_proposal_edit
+      comments_body_max_length
+      proposal_code_prefix
+      votes_for_proposal_success
+      months_to_archive_proposals
+      email_domain_for_officials
+      facebook_handle
+      instagram_handle
+      telegram_handle
+      twitter_handle
+      twitter_hashtag
+      youtube_handle
+      org_name
+      meta_title
+      meta_description
+      meta_keywords
+      proposal_notification_minimum_interval_in_days
+      direct_message_max_per_day
+      mailer_from_name
+      mailer_from_address
+      min_age_to_participate
+      hot_score_period_in_days
+      related_content_score_threshold
+      featured_proposals_number
+      feature.dashboard.notification_emails
```

Comment:
Hi @javierm! Thanks.

What do you think about moving this setting to the proposals dashboard tab? Maybe it is easier to find by users if we move it to this tab.

<img width="1680" alt="Captura de pantalla 2024-01-19 a las 10 53 00" src="https://github.com/consuldemocracy/consuldemocracy/assets/15726/dee2ff43-23f4-4940-9126-985e960130e2">

Reply:
There are other settings like this one that we can move to their very specific tab:

* feature.map
* feature.allow_attached_documents
* feature.allow_images
* feature.remote_census
* feature.sdg

Reply:
cc @taitus

Reply:
While we decide, I place it in the features tab.

Reply:
I'm fine with moving `feature.dashboard.notification_emails`, although maybe we could do it in a different pull request :thinking:. I'm approving this one as it is.

Regarding `feature.remote_census` and `feature.sdg` are a bit tricky because the content of the remote census tab and SDG configuration tab depend on them, so we'd need to change those tabs a little bit :thinking:. I'm fine with changing them, but I'd rather wait until we release version 2.1.

I don't know about the rest right now; we can have a look.

Reply:
Yes. No problem, we can do it for the next release.